### DISCUSSION TITLE
Auto-hide mouse cursor

### DISCUSSION THREAD

**Discussion by DusanLesan** [hyprwm/Hyprland#260] [Q&A] [answered]
I would like for my cursor to be hidden unless there is actual mouse activity detected. 
On the X, I used app unclutter but here I see no alternative. How to hide mouse pointer?

As I do not even have a mouse, I would like for window splitting not to be done under the cursor location. I want it to be in down-right position always. Or even better, to be master stack layout. How to do this?

**Comment by vaxerski** [comment, marked as answer]
master-stack is on the bucket list but not implemented yet.

For dwindle controls, including forcing the split etc etc see the [dwindle wiki page](https://github.com/hyprwm/Hyprland/wiki/Dwindle-Layout) (reading the wiki usually helps)

For hiding the cursor, an option is not implemented for that, but I can add it.

**Reply by vaxerski** [reply]
added `general:cursor_inactive_timeout` in 72c86b99eb87b23c11fdc795ec6f3668f283b95c

**Reply by DusanLesan** [reply]
Timer works great. Thank you.
I'll read the doc about dwindle and I'll look forward for master-stack.

**Reply by DusanLesan** [reply]
Actually there is a issue with it. It disappears after some time but it is displayed again after spawning new windows. After that, it is stuck in on state

**Reply by vaxerski** [reply]
try with the last commit

**Reply by DusanLesan** [reply]
That is it. Awesome!

**Reply by bearcatsandor** [reply]
Thanks folks, that's great. I was missing this from Sway.
